src/ui/components/Planner/WeeklyPlanner/utils.ts

```diff
@@ -0,0 +1,113 @@
+import { DayDetailType, IODataStructure, WeeklyPlannerDataType } from "interface/WeeklyPlannerTypes";
+import { convertTime, generateDateFromStringInput, getFrenchDayFromDate } from "../../../utils";
```

Comment:
FrenchDay is harcoded here, could be a problem for reuse outside France, complicated to have a more generic way of handling formats ?